frontend/eslint.config.mjs

```diff
@@ -0,0 +1,72 @@
+import { fixupConfigRules, fixupPluginRules } from '@eslint/compat'
+import typescriptEslint from '@typescript-eslint/eslint-plugin'
+import prettier from 'eslint-plugin-prettier'
+import simpleImportSort from 'eslint-plugin-simple-import-sort'
+import cypress from 'eslint-plugin-cypress'
+import tsParser from '@typescript-eslint/parser'
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+import js from '@eslint/js'
+import { FlatCompat } from '@eslint/eslintrc'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+})
+
+export default [
+  ...fixupConfigRules(
+    compat.extends(
+      'eslint:recommended',
+      'plugin:@typescript-eslint/eslint-recommended',
+      'plugin:@typescript-eslint/recommended',
+      'plugin:react/recommended',
+      'plugin:react/jsx-runtime',
+      'plugin:react-hooks/recommended',
+      'next/core-web-vitals',
+    ),
+  ),
+  {
+    plugins: {
+      '@typescript-eslint': fixupPluginRules(typescriptEslint),
+      prettier,
+      'simple-import-sort': simpleImportSort,
+      cypress,
+    },
+
+    languageOptions: {
+      parser: tsParser,
+    },
+
+    rules: {
+      '@typescript-eslint/no-unused-vars': [
+        'warn',
+        {
+          argsIgnorePattern: '^_',
+          varsIgnorePattern: '^_',
+          caughtErrorsIgnorePattern: '^_',
+        },
+      ],
+
+      '@typescript-eslint/no-extra-semi': 'off',
+      '@typescript-eslint/no-explicit-any': 'off',
+      '@typescript-eslint/no-require-imports': 'off',
+      '@typescript-eslint/no-unused-expressions': 'off',
+
+      'react/jsx-newline': [
+        1,
+        {
+          prevent: true,
+        },
+      ],
+
+      'cypress/no-async-tests': 'off',
```

Comment:
Our Cypress suite has its own eslint config. This should go in there.

On that note, I don't see an updated version of our Cypress eslint config in this PR.